ci-automation/packages-tag.sh

```diff
@@ -98,9 +98,12 @@ function _packages_tag_impl() {
           local ret=0
           git diff --exit-code "${existing_tag}" || ret=$?
           if [[ ret -eq 0 ]]; then
-            touch ./skip-build
-            echo "Creating ./skip-build flag file, indicating that the build must not to continue because no new tag got created as there are no changes since tag ${existing_tag}" >&2
-            return 0
+            if curl --head --fail --silent --show-error --location "https://${BUILDCACHE_SERVER}/images/amd64/${FLATCAR_VERSION}/flatcar_production_image.bin.bz2"; then
```

Comment:
Should we maybe check both amd64 and arm64 images here?

Reply:
I can also do the same check in the sdk-bootstrap then to let it retry to build the image if that failed in the last nightly for some reason.

Reply:
Updated it now to prefer a rebuild if something is missing